moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -425,9 +425,9 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     return asyncExecute(plan) == MoveItErrorCode::SUCCESS;
   }
 
-  const char* getPlannerIdCStr() const
```

Comment:
Oops. This became an infinite recursive function call now. Where is this function (or the previous `getPlannerIdCStr`) used at all?

Reply:
I think we tried to fix something that git already did, or perhaps it was on an outdated diff. This conversation seems to be on an outdated diff as well... :/ 

The current diff shows me adding this function which is absolutely not intended. Big of a jump from a PR intended for Kinetic a year ago :)

Reply:
The github diff view seems to be cached and I discovered a refresh button on the top... That may be the trick. I've added a new commit to remove the function, compilation seems fine, let's see travis with the tests.